app/hotels/src/singleHotel/hotelInformation/description/Facilities.js

```diff
@@ -42,8 +42,8 @@ export class Facilities extends React.Component<Props, State> {
   render() {
     const { facilities } = this.props;
     const { collapsed } = this.state;
-    const edges = idx(facilities, _ => _.edges) || [];
-    const fullList = edges.map(edge => idx(edge, _ => _.node));
+    const fullList = idx(facilities, _ => _.amenities) || [];
```

Comment:
It's weird that `amenities` should live under `facilities` object. I know I am being difficult with vocabulary, but I think it is for the best in the long run.

Reply:
You are right, I will fix this 😊